src/lib/movies.ts

```diff
@@ -106,6 +106,29 @@ export const getMovies = unstable_cache(
   { revalidate: 1 },
 );
 
+export const getTheaters = unstable_cache(
+  async (): Promise<string[]> => {
+    try {
+      const { db } = getFirebase();
+      const docRef = doc(db, "website-extra-docs", "theaters_names");
```

Comment:
When is `theaters_names` generated? Can you add the code that you used to generate it to a PR in `scraper`? Also, can we rename this document to `all-theaters`, with inside `elements`? (Just for consistency)